Configuration/PyReleaseValidation/python/upgradeWorkflowComponents.py

```diff
@@ -342,6 +342,39 @@ def condition_(self, fragment, stepList, key, hasHarvest):
 }
 # end of Patatrack workflows
 
+class UpgradeWorkflow_Run3ProdLike(UpgradeWorkflow):
```

Comment:
@fabiocos couldn't we remove the need for an extra class and just use the existing ProdLike special workflow?

I think you would just need to add 'RecoFull' and 'DigiFull' to its list of steps (along with any associated setup operations), and modify the condition to allow for 2021 as well as 2026.

Reply:
@kpedro88 yes, I was thinking to this possibility as well once agreed on the proposal, there is a bit of gymnastic involved to avoid interferences between Run3 and Phase2 but it is not that complex.